templates/docker-rootful.yaml

```
@@ -68,3 +68,7 @@ message: |
   docker context use lima-{{.Name}}
   docker run hello-world
   ------
+portMonitors:
+  docker:
+    sockets:
+    - "/var/run/docker.sock"
```

Nitpicking, but move this block below the `portForwards` section to keep `message` at the very bottom.

Also the filename doesn't need to be quoted unless it contains template variables:

```suggestion
portMonitors:
  docker:
    sockets:
    - /var/run/docker.sock
```